test/eth/Cdp.spec.js

```diff
@@ -182,21 +204,39 @@ const sharedTests = openCdp => {
       });
 
       describe('with drip', () => {
-        let snapshotId;
+        let snapshotId, nonceService;
+
+        beforeAll(() => {
+          transactionCount = cdpService
```

Comment:
you can use the `nonceService` defined in the next line and set `transactionCount = nonceService._counts[currentAccount`